lib/biran/configurinator.rb

```diff
@@ -73,7 +73,7 @@ def build_db_config
 
     def base_db_config
       return @base_db_config if @base_db_config
-      return @base_db_config = {} unless File.exists? default_db_config_file
+      return @base_db_config = {} unless File.exist? default_db_config_file
```

Comment:
This was the only change needed at this time. There might be some other depreciated things we can look at, but this was needed to have this gem work on 3.2+